instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import Comment from "./Comment";
+import PropTypes from "prop-types";
+import AddComment from "./AddComment";
+import "./CommentSection.css";
+
+class CommentSection extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      comments: props.user.comments,
+      timestamp: props.user.timestamp,
```

Comment:
not clear why this is on `state`, since it doesn't change. Only put properties that would change on `state`